module/Activity/view/activity/activity/create.phtml

```diff
@@ -5,9 +5,10 @@ $this->headLink()
 $this->headScript()
     ->prependFile($this->basePath() . '/js/activity-create.js')
     ->prependFile($this->basePath() . '/js/jquery.datetimepicker.full.min.js');
-$form->prepare();
+if (!$success) {$form->prepare();}
 ?>
 <div class="container">
+    <?php if (!$success): ?>
```

Comment:
Having two separate templates is indeed a cleaner solution.

Note that the success page is probably only temporary, since, after more organiser features are implemented (e.g. editing after submitting), redirecting to the organiser page on success makes more sense.